libraries/aws/provisioning/include/aws_iot_provisioning.h

```diff
@@ -92,6 +92,21 @@ AwsIotProvisioningError_t AwsIotProvisioning_Init( uint32_t mqttTimeout );
  * functions can result in undefined behavior. Device provisioning with this library
  * REQUIRES calling the API functions of this library sequentially.
  *
+ * @warning Do not overwrite the existing Provisioning claim credentials with the new
+ * credentials provided by the server, at least until the device has been provisioned
+ * with a new certificate using the @ref provisioning_function_registerthing function.
+ * It is also recommended to always retain the Provisioning claim credentials,
+ * if the claim certificate is long-lived.
```

Comment:
What does long-lived claim certificate mean? Is that a standard?

Reply:
I think if we don't provide a clear definition of what a long lived certificate means, that would be confusing to the reader. For an example a reader could interpret the long -lived certificate as certificate that is valid for 30 days. What do you think?

Reply:
In this use-case of "Provisioning By Claim", the claim credential can be created by the customer and registered on AWS IoT, thus, specifying a duration for "long-lived" certificate is not suitable here. 
The intent of the statement is to suggest that customer should retain the Provisioning Claim if they have the product use-case for re-provisioning the device with the same certificate (which implies that the certificate be valid to support re-provisioning). Maybe, I can re-word the sentence to say that "It is also recommended to always retain the Provisioning claim credentials, if your product use-case supports re-provisioning of the device"